diff --git a/modules/custom/old_bay/src/Controller/OldBayController.php b/modules/custom/old_bay/src/Controller/OldBayController.php
index e010e28..5a513c4 100644
--- a/modules/custom/old_bay/src/Controller/OldBayController.php
+++ b/modules/custom/old_bay/src/Controller/OldBayController.php
@@ -7,14 +7,6 @@
 
 class OldBayController extends ControllerBase {
   public function seasonFood($count) {
-    if ($count === null) {
-      $configFactory = \Drupal::getContainer()
-        ->get('config.factory');
-
-      $count = $configFactory->get('old_bay.default')
-        ->get('default_count');
-    }
-
     $seasoner = \Drupal::getContainer()
       ->get('old_bay.seasoner');
     $foods = $seasoner->seasonSomeRandomFood($count);
diff --git a/modules/custom/old_bay/src/Service/OldBaySeasoner.php b/modules/custom/old_bay/src/Service/OldBaySeasoner.php
index e8205b3..511a733 100644
--- a/modules/custom/old_bay/src/Service/OldBaySeasoner.php
+++ b/modules/custom/old_bay/src/Service/OldBaySeasoner.php
@@ -4,6 +4,14 @@
 
 class OldBaySeasoner {
   public function seasonSomeRandomFood($count) {
+    if ($count === null) {
+      $configFactory = \Drupal::getContainer()
+        ->get('config.factory');
+
+      $count = $configFactory->get('old_bay.default')
+        ->get('default_count');
+    }
+
     $foods = ['popcorn', 'french fries', 'soup', 'chicken'];
     $seasonedFoods = [];
     for ($i = 0; $i <= $count; $i++) {
