diff --git a/modules/custom/old_bay/src/Service/OldBaySeasoner.php b/modules/custom/old_bay/src/Service/OldBaySeasoner.php
index a279dbb..07c692d 100644
--- a/modules/custom/old_bay/src/Service/OldBaySeasoner.php
+++ b/modules/custom/old_bay/src/Service/OldBaySeasoner.php
@@ -2,10 +2,19 @@
 
 namespace Drupal\old_bay\Service;
 
+use Drupal\Core\Config\ConfigFactoryInterface;
+
 class OldBaySeasoner {
+
+  private $configFactory;
+
+  public function __construct(ConfigFactoryInterface $configFactory) {
+    $this->configFactory = $configFactory;
+  }
+
   public function seasonSomeRandomFood($count) {
     if ($count === null) {
-      $count = $this->config('old_bay.default')
+      $count = $this->configFactory->get('old_bay.default')
         ->get('default_count');
     }
 
