diff --git a/modules/custom/old_bay/old_bay.services.yml b/modules/custom/old_bay/old_bay.services.yml
index ffdf531..1023c7b 100644
--- a/modules/custom/old_bay/old_bay.services.yml
+++ b/modules/custom/old_bay/old_bay.services.yml
@@ -2,4 +2,8 @@ services:
   old_bay.seasoner:
     class: Drupal\old_bay\Service\OldBaySeasoner
     # arguments: ['@config.factory']
-    autowire: true
\ No newline at end of file
+    autowire: true
+
+  old_bay.event_subscriber:
+    class: Drupal\old_bay\EventSubscriber\OldBayEventSubscriber
+    arguments: []
diff --git a/modules/custom/old_bay/src/EventSubscriber/OldBayEventSubscriber.php b/modules/custom/old_bay/src/EventSubscriber/OldBayEventSubscriber.php
index 4e30c6f..0649eae 100644
--- a/modules/custom/old_bay/src/EventSubscriber/OldBayEventSubscriber.php
+++ b/modules/custom/old_bay/src/EventSubscriber/OldBayEventSubscriber.php
@@ -2,6 +2,16 @@
 
 namespace Drupal\old_bay\EventSubscriber;
 
-class OldBayEventSubscriber {
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
 
-}
\ No newline at end of file
+class OldBayEventSubscriber implements EventSubscriberInterface {
+  public function onKernelRequest() {
+    drupal_set_message('Grab the old bay! We are cooking!');
+  }
+
+  public static function getSubscribedEvents() {
+    return [
+      'kernel.request' => 'onKernelRequest'
+    ];
+  }
+}
