diff --git a/modules/custom/old_bay/old_bay.services.yml b/modules/custom/old_bay/old_bay.services.yml
index 76ce920..4817d3e 100644
--- a/modules/custom/old_bay/old_bay.services.yml
+++ b/modules/custom/old_bay/old_bay.services.yml
@@ -9,3 +9,8 @@ services:
     arguments: []
     tags:
       - { name: event_subscriber }
+  old_bay.route_subscriber:
+    class: Drupal\old_bay\Routing\RouteSubscriber
+    tags:
+      - { name: event_subscriber }
+
diff --git a/modules/custom/old_bay/src/Routing/RouteSubscriber.php b/modules/custom/old_bay/src/Routing/RouteSubscriber.php
new file mode 100644
index 0000000..6058cb7
--- /dev/null
+++ b/modules/custom/old_bay/src/Routing/RouteSubscriber.php
@@ -0,0 +1,20 @@
+<?php
+
+namespace Drupal\old_bay\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Class RouteSubscriber.
+ *
+ * @package Drupal\old_bay\Routing
+ * Listens to the dynamic route events.
+ */
+class RouteSubscriber extends RouteSubscriberBase {
+  /**
+   * {@inheritdoc}
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+  }
+}
