diff --git a/modules/custom/old_bay/old_bay.info.yml b/modules/custom/old_bay/old_bay.info.yml
new file mode 100644
index 0000000..097ca0b
--- /dev/null
+++ b/modules/custom/old_bay/old_bay.info.yml
@@ -0,0 +1,5 @@
+name: old_bay
+type: module
+description: Old Bay Seasoning woohoo!
+core: 8.x
+package: Custom
diff --git a/modules/custom/old_bay/old_bay.module b/modules/custom/old_bay/old_bay.module
new file mode 100644
index 0000000..f810dc4
--- /dev/null
+++ b/modules/custom/old_bay/old_bay.module
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file
+ * Contains old_bay.module..
+ */
+
+use Drupal\Core\Routing\RouteMatchInterface;
+
+/**
+ * Implements hook_help().
+ */
+function old_bay_help($route_name, RouteMatchInterface $route_match) {
+  switch ($route_name) {
+    // Main module help for the old_bay module.
+    case 'help.page.old_bay':
+      $output = '';
+      $output .= '<h3>' . t('About') . '</h3>';
+      $output .= '<p>' . t('Old Bay Seasoning woohoo!') . '</p>';
+      return $output;
+
+    default:
+  }
+}
