diff --git a/modules/custom/old_bay/src/Plugin/Block/FoodsBlock.php b/modules/custom/old_bay/src/Plugin/Block/FoodsBlock.php
index ff27fdf..538579d 100644
--- a/modules/custom/old_bay/src/Plugin/Block/FoodsBlock.php
+++ b/modules/custom/old_bay/src/Plugin/Block/FoodsBlock.php
@@ -20,7 +20,10 @@ class FoodsBlock extends BlockBase {
    */
   public function build() {
     $build = [];
-    $build['foods_block']['#markup'] = 'Implement FoodsBlock.';
+    $foods = \Drupal::getContainer()
+      ->get('old_bay.seasoner')
+      ->seasonSomeRandomFood(5);
+    $build['foods_block']['#markup'] = implode('<br/>', $foods);
 
     return $build;
   }
