diff --git a/modules/custom/old_bay/old_bay.routing.yml b/modules/custom/old_bay/old_bay.routing.yml
index 6397974..6c81d03 100644
--- a/modules/custom/old_bay/old_bay.routing.yml
+++ b/modules/custom/old_bay/old_bay.routing.yml
@@ -1,4 +1,6 @@
 old_bay_season_food:
-  path: /season/oldbay/popcorn
+  path: /season/oldbay
+  defaults:
+    _controller: Drupal\old_bay\Controller\OldBayController::seasonFood
   requirements:
     _permission: 'access content'
diff --git a/modules/custom/old_bay/src/Controller/OldBayController.php b/modules/custom/old_bay/src/Controller/OldBayController.php
new file mode 100644
index 0000000..9a6a1d4
--- /dev/null
+++ b/modules/custom/old_bay/src/Controller/OldBayController.php
@@ -0,0 +1,11 @@
+<?php
+
+namespace Drupal\old_bay\Controller;
+
+use Symfony\Component\HttpFoundation\Response;
+
+class OldBayController {
+  public function seasonFood() {
+    return new Response('Boom! Your popcorn got old bayified!');
+  }
+}
