diff --git a/modules/custom/old_bay/config/install/old_bay.default.yml b/modules/custom/old_bay/config/install/old_bay.default.yml
new file mode 100644
index 0000000..c62aa26
--- /dev/null
+++ b/modules/custom/old_bay/config/install/old_bay.default.yml
@@ -0,0 +1 @@
+old_bay:
diff --git a/modules/custom/old_bay/old_bay.links.menu.yml b/modules/custom/old_bay/old_bay.links.menu.yml
new file mode 100644
index 0000000..38567dc
--- /dev/null
+++ b/modules/custom/old_bay/old_bay.links.menu.yml
@@ -0,0 +1,7 @@
+old_bay.default_form:
+  title: 'Old Bay Settings'
+  route_name: old_bay.default_form
+  description: 'A description for the menu entry'
+  parent: system.admin_config_system
+  weight: 99
+
diff --git a/modules/custom/old_bay/old_bay.routing.yml b/modules/custom/old_bay/old_bay.routing.yml
index cb543b3..69bb22c 100644
--- a/modules/custom/old_bay/old_bay.routing.yml
+++ b/modules/custom/old_bay/old_bay.routing.yml
@@ -5,3 +5,14 @@ old_bay_season_food:
     count: 50
   requirements:
     _permission: 'access content'
+
+old_bay.default_form:
+  path: '/admin/config/old_bay/default'
+  defaults:
+    _form: '\Drupal\old_bay\Form\DefaultForm'
+    _title: 'DefaultForm'
+  requirements:
+    _permission: 'access administration pages'
+  options:
+    _admin_route: TRUE
+
diff --git a/modules/custom/old_bay/src/Form/DefaultForm.php b/modules/custom/old_bay/src/Form/DefaultForm.php
new file mode 100644
index 0000000..8a2c143
--- /dev/null
+++ b/modules/custom/old_bay/src/Form/DefaultForm.php
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\old_bay\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Class DefaultForm.
+ *
+ * @package Drupal\old_bay\Form
+ */
+class DefaultForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return [
+      'old_bay.default',
+    ];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'default_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('old_bay.default');
+    $form['default_count'] = [
+      '#type' => 'number',
+      '#title' => $this->t('Default Count'),
+      '#description' => $this->t('Default number of foods to old bayify'),
+      '#default_value' => $config->get('default_count'),
+    ];
+    return parent::buildForm($form, $form_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function validateForm(array &$form, FormStateInterface $form_state) {
+    parent::validateForm($form, $form_state);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+    parent::submitForm($form, $form_state);
+
+    $this->config('old_bay.default')
+      ->set('default_count', $form_state->getValue('default_count'))
+      ->save();
+  }
+
+}
