diff --git a/modules/custom/old_bay/src/Controller/OldBayController.php b/modules/custom/old_bay/src/Controller/OldBayController.php
index c1751e0..61c556d 100644
--- a/modules/custom/old_bay/src/Controller/OldBayController.php
+++ b/modules/custom/old_bay/src/Controller/OldBayController.php
@@ -3,6 +3,7 @@
 namespace Drupal\old_bay\Controller;
 
 use Drupal\Core\Controller\ControllerBase;
+use Drupal\old_bay\Service\OldBaySeasoner;
 
 class OldBayController extends ControllerBase {
   public function seasonFood($count) {
@@ -11,8 +12,11 @@ public function seasonFood($count) {
         ->get('default_count');
     }
 
+    $seasoner = new OldBaySeasoner();
+    $foods = $seasoner->seasonSomeRandomFood($count);
+
     $message = sprintf(
-      'Boom! We just bayified %s pieces of popcorn!',
+      'Boom! Dinner is serviced: <br/>'.implode("<br/>", $foods),
       $count
     );
 
diff --git a/modules/custom/old_bay/src/Service/OldBaySeasoner.php b/modules/custom/old_bay/src/Service/OldBaySeasoner.php
new file mode 100644
index 0000000..8368509
--- /dev/null
+++ b/modules/custom/old_bay/src/Service/OldBaySeasoner.php
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\old_bay\Service;
+
+class OldBaySeasoner {
+  public function seasonSomeRandomFood($count) {
+    $foods = ['popcorn', 'french fries', 'soup', 'chicken'];
+    $seasonedFoods = [];
+    for ($i = 0; $i < $count; $i++) {
+      $foodKey = array_rand($foods);
+      $seasonedFoods[] = 'Bayified some '.$foods[$foodKey];
+    }
+
+    return $seasonedFoods;
+  }
+}
