diff --git a/modules/custom/old_bay/src/Controller/OldBayController.php b/modules/custom/old_bay/src/Controller/OldBayController.php
index c1751e0..94be67b 100644
--- a/modules/custom/old_bay/src/Controller/OldBayController.php
+++ b/modules/custom/old_bay/src/Controller/OldBayController.php
@@ -7,7 +7,10 @@
 class OldBayController extends ControllerBase {
   public function seasonFood($count) {
     if ($count === null) {
-      $count = $this->config('old_bay.default')
+      $configFactory = \Drupal::getContainer()
+        ->get('config.factory');
+
+      $count = $configFactory->get('old_bay.default')
         ->get('default_count');
     }
 
