diff --git a/modules/custom/old_bay/old_bay.routing.yml b/modules/custom/old_bay/old_bay.routing.yml
index 69bb22c..49fcd15 100644
--- a/modules/custom/old_bay/old_bay.routing.yml
+++ b/modules/custom/old_bay/old_bay.routing.yml
@@ -2,7 +2,7 @@ old_bay_season_food:
   path: /season/oldbay/{count}
   defaults:
     _controller: Drupal\old_bay\Controller\OldBayController::seasonFood
-    count: 50
+    count: null
   requirements:
     _permission: 'access content'
 
diff --git a/modules/custom/old_bay/src/Controller/OldBayController.php b/modules/custom/old_bay/src/Controller/OldBayController.php
index 5013496..c1751e0 100644
--- a/modules/custom/old_bay/src/Controller/OldBayController.php
+++ b/modules/custom/old_bay/src/Controller/OldBayController.php
@@ -2,8 +2,15 @@
 
 namespace Drupal\old_bay\Controller;
 
-class OldBayController {
+use Drupal\Core\Controller\ControllerBase;
+
+class OldBayController extends ControllerBase {
   public function seasonFood($count) {
+    if ($count === null) {
+      $count = $this->config('old_bay.default')
+        ->get('default_count');
+    }
+
     $message = sprintf(
       'Boom! We just bayified %s pieces of popcorn!',
       $count
